notebooks/228-clip-zero-shot-image-classification/228-clip-quantize-nncf.ipynb

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    fp16_model_path = 'clip-vit-base-patch16.xml'
</pre><p>should we chack that this file really exists?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>